docs/operations/hotplug_interfaces.md

```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+>**Note**: This step can be done when both VM and VMI objects has the new interface in their spec.
```

Comment:
I do not understand this note.
Is it related to [this comment](https://github.com/kubevirt/user-guide/pull/690/files#r1241747034)?